uint32 sonar_id
---
uint32 value_cm
